src/eko/io/runcards.py

```diff
@@ -49,6 +49,8 @@ class TheoryCard(DictLike):
     """Ratio between factorization scale and process scale."""
     n3lo_ad_variation: N3LOAdVariation
     """|N3LO| anomalous dimension variation: ``(gg_var, gq_var, qg_var, qq_var)``."""
+    matching_order: Order
```

Comment:
Maybe here it would even be fine to have a default
```suggestion
    matching_order: Optional[Order] = None
```

Such that by default we consider it equal to the evolution order.
(instead of doing it only for the old-style runcards)

Reply:
I'm fine with and without default. @felixhekhorn are you happy with the dafult?

Reply:
I'm fine with either ... as I always say eventually it may turn out impossible to keep "no defaults" and stable API - this is a clear example why: it is backward compatible, but we introduce defaults or it is a breaking change with no defaults

Reply:
well but here the default can also make sense, let's go for it.